src/main/java/step2/StringCalculator.java

```diff
@@ -0,0 +1,43 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    public int addOperation(String text) {
+        if (text == null)
```

Comment:
코드 컨벤션이 통일 되면 더 좋을 것 같습니다 😄 
```suggestion
        if (text == null) {
```